### PR TITLE
 [bug 1155107] Filter against BlockList. 

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@alexgibson r?

**Comment by alexgibson** [comment]
Sorry I haven't got round to looking at this just yet, will try and do so tomorrow

File: docs/developing.rst [hunk 1]
```diff
@@ -172,6 +172,70 @@ The syntax in a snippet is slightly different and uses square brackets `[[snippe
...
+       Foo! <a href="#" id="block-snippet-link">Do not show again</a>
+     </div>
+     <script type="text/javascript">
+         //<![CDATA[
+         (function() {
+             var snippet = document.getElementById('block-snippet-[[snippet_id]]');
+             snippet.addEventListener('show_snippet', function() {
+                 (function () {
+                 var link = document.getElementById('block-snippet-link');
+                  link.onclick = function() {
```
Comment (alexgibson) [inline docs/developing.rst]: Nit: fix indentation

**Comment by alexgibson** [comment]
Code looks fine to me, r+wc :cactus:

**Comment by glogiotatidis** [comment]
Thanks Alex :)

**Pull request merged**